src/hb-buffer-serialize.cc

```diff
@@ -508,18 +525,25 @@ hb_buffer_serialize_glyphs (hb_buffer_t *buffer,
  * The serialized codepoints will look something like:
  *
  * ```
- * <U+0651|U+0628>
+ * <U+0651=0|U+0628=1>
  * ```
  * - Glyphs are separated with `|`
  * - Unicode codepoints are expressed as zero-padded four (or more)
  *   digit hexadecimal numbers preceded by `U+`
+ * - If #HB_BUFFER_SERIALIZE_FLAG_NO_CLUSTERS is not set, the cluster
+ *   will be indicated with a `=` then #hb_glyph_info_t.cluster.
  *
  * ## json
  * A machine-readable, structured format.
- * The serialized codepoints will be a list of Unicode codepoints as
- * decimal integers. For example:
+ * The serialized codepoints will be a list of objects with the following
+ * properties:
+ * - `g`: the Unicode codepoint as a decimal integer
```

Comment:
Use of `g` here is confusing, shouldn’t be `u` or `c`?

Reply:
Sorry, yes, typo. (u)